### DISCUSSION TITLE
How can I use andes in HELICS?

### DISCUSSION THREAD

**Discussion by cqss11** [GMLC-TDC/HELICS#2546] [General]
I only find the examples of GridLAB-D in HELICS.
But I want to use andes in HELICS to run co-simulation. How can I do it?
Please let me know.Thanks.

**Comment by nightlark** [comment]
Is [andes](https://github.com/curent/andes) the simulator you are referring to?

As far as I'm aware, no one has written any form of HELICS integration for andes yet. It looks like andes is written in python, so using the HELICS Python interface ([pyhelics](https://github.com/GMLC-TDC/pyhelics)) would be the place to start -- from there it will likely involve modifying andes (or making use of functions andes provides for controlling time advancement) so that HELICS is able to control time advancement.

https://docs.helics.org/en/latest/user-guide/fundamental_topics/simulator_integration.html in the documentation on some of the functions in HELICS that are likely to be useful for adding support for a new simulator.
